EntraIdUsersSupportQuickstart/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Azure.Identity;
+using Azure.Communication;
+
+namespace EntraIdUsersSupportQuickstart
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            Console.WriteLine("Azure Communication Services - Obtain Access Token for Entra ID User Quickstart");
+
+            // This code demonstrates how to fetch your Microsoft Entra client ID and tenant ID from environment variables.
+            string clientId = Environment.GetEnvironmentVariable("ENTRA_CLIENT_ID");
+            string tenantId = Environment.GetEnvironmentVariable("ENTRA_TENANT_ID");
+
+            //Initialize InteractiveBrowserCredential for use with CommunicationTokenCredential.
+            var options = new InteractiveBrowserCredentialOptions
+            {
+                TenantId = tenantId,
+                ClientId = clientId,
+            };
+            var entraTokenCredential = new InteractiveBrowserCredential(options);

```

Comment:
Let's use some more succinct and modern C# style, I think this works (please make sure it does compile)
No need to have extra variables for the Option arguments

```suggestion
            InteractiveBrowserCredential entraTokenCredential = new (new {
               TenantId: tenantId,
               ClientId: clientID
            });
```

Reply:
need to define the class 
<img width="815" alt="image" src="https://github.com/user-attachments/assets/f76b9716-122f-4795-8757-028b0d643ce2" />